validator/client/src/main/java/tech/pegasys/teku/validator/client/AbstractDutyScheduler.java

```diff
@@ -174,15 +174,26 @@ protected void onProductionDue(final UInt64 slot) {
       return;
     }
 
-    if (!isAbleToVerifyEpoch(slot)) {
-      LOG.info(
-          "Not performing {} duties for slot {} because it is too far ahead of the current slot {}",
-          dutyType,
+    if (lastProductionSlot != null && slot.compareTo(lastProductionSlot) == 0) {
+      LOG.debug(
+          "{} duties for slot {} already performed, last production slot {}",
+          dutyType.substring(0, 1).toUpperCase() + dutyType.substring(1),
           slot,
-          getCurrentEpoch().map(UInt64::toString).orElse("UNDEFINED"));
+          lastProductionSlot);
+      return;
+    }
+
+    if (!isAbleToVerifyEpoch(slot)) {
+      LOG.info("Not performing {} duties for slot {}, unable to verify its epoch", dutyType, slot);
```

Comment:
The original wording was probably more informative here.  We are unable to verify the epoch because the slot is too far ahead.  It is somewhat confusing that `currentEpoch` may be unknown initially but I think it's still important to show it's value so the message can be properly understood.  Possibly we should just omit logging the message if `currentEpoch` is empty since that's routine.